app/services/PropertyService.scala

```diff
@@ -713,4 +725,29 @@ class PropertyService @Inject() (connector: IntegrationFrameworkConnector, repos
     val annualUkOtherProperty = AnnualUkOtherProperty(None, Some(allowances))
     PropertyAnnualSubmission(None, None, None, None, Some(annualUkOtherProperty))
   }
+
+  private def getPropertySubmissionForRentARoomAllowances(answers: RentARoomAllowances) = {
+    val allowances = UkOtherAllowances(
+      answers.annualInvestmentAllowance,
+      answers.zeroEmissionGoodsVehicleAllowance,
+      answers.businessPremisesRenovationAllowance,
+      answers.otherCapitalAllowance,
+      answers.replacementOfDomesticGoodsAllowance,
+      answers.electricChargePointAllowance.flatMap(_.electricChargePointAllowanceAmount),
+      None,
+      None,
+      answers.zeroEmissionCarAllowance,
+      None
+    )
+
+    val newAllowance = answers.capitalAllowancesForACar.fold{
+      allowances
+    }{
+      amount => allowances.copy(otherCapitalAllowance = amount.capitalAllowancesForACarAmount)
+    }
+
+    val annualUkOtherProperty = AnnualUkOtherProperty(None, Some(newAllowance))
```

Comment:
Lets not set other values None, it will override values set by  other sections or third party softwares 

Fix: Use Monocle library

Reply:
Done! I would argue though that if depth = 1 in a case class copy works fine and Monocle is a bit of an over-engineering

Reply:
@arunkumar461 I agree that we should use None when we want to test the behavior that value is absent and at other times have some data in it. Lets discuss tomorrow after SU